packages/example-log-extension/src/types.ts

```diff
@@ -7,6 +7,9 @@
 
 import {ParsedRequest, OperationArgs} from '@loopback/rest';
 
+/**
+ * A function to perform REST req/res logging action
+ */
 export interface LogFn {
```

Comment:
This is out of sync with README. README uses `LogAction`, the source code here uses `LogFn`. Please pick one name and use it consistently everywhere.

Reply:
Fixed